2022-05-11 04:37:44,427 INFO    MainThread:31249 [wandb_setup.py:_flush():69] setting env: {'run_group': 'dev', 'project': 'exercise_14'}
2022-05-11 04:37:44,427 INFO    MainThread:31249 [wandb_setup.py:_flush():69] setting login settings: {}
2022-05-11 04:37:44,428 INFO    MainThread:31249 [wandb_init.py:_log_setup():334] Logging user logs to /mnt/d/Udacity/MLOps/Udacity Exercises/nd0821-c2-build-model-workflow-exercises/lesson-5-final-pipeline-release-and-deploy/exercises/exercise_14/solution/segregate/wandb/run-20220511_043744-sznd9ciq/logs/debug.log
2022-05-11 04:37:44,428 INFO    MainThread:31249 [wandb_init.py:_log_setup():335] Logging internal logs to /mnt/d/Udacity/MLOps/Udacity Exercises/nd0821-c2-build-model-workflow-exercises/lesson-5-final-pipeline-release-and-deploy/exercises/exercise_14/solution/segregate/wandb/run-20220511_043744-sznd9ciq/logs/debug-internal.log
2022-05-11 04:37:44,429 INFO    MainThread:31249 [wandb_init.py:init():367] calling init triggers
2022-05-11 04:37:44,429 INFO    MainThread:31249 [wandb_init.py:init():372] wandb.init called with sweep_config: {}
config: {}
2022-05-11 04:37:44,430 INFO    MainThread:31249 [wandb_init.py:init():416] starting backend
2022-05-11 04:37:44,430 INFO    MainThread:31249 [backend.py:_multiprocessing_setup():69] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-05-11 04:37:44,439 INFO    MainThread:31249 [backend.py:ensure_launched():123] starting backend process...
2022-05-11 04:37:44,442 INFO    MainThread:31249 [backend.py:ensure_launched():127] started backend process with pid: 31269
2022-05-11 04:37:44,443 INFO    MainThread:31249 [wandb_init.py:init():421] backend started and connected
2022-05-11 04:37:44,446 INFO    MainThread:31249 [wandb_init.py:init():461] updated telemetry
2022-05-11 04:37:44,446 INFO    MainThread:31249 [wandb_init.py:init():480] communicating current version
2022-05-11 04:37:44,954 INFO    MainThread:31249 [wandb_init.py:init():485] got version response upgrade_message: "wandb version 0.12.16 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2022-05-11 04:37:44,955 INFO    MainThread:31249 [wandb_init.py:init():493] communicating run to backend with 30 second timeout
2022-05-11 04:37:45,349 INFO    MainThread:31249 [wandb_init.py:init():518] starting run threads in backend
2022-05-11 04:37:45,448 INFO    MainThread:31249 [wandb_run.py:_console_start():1476] atexit reg
2022-05-11 04:37:45,451 INFO    MainThread:31249 [wandb_run.py:_redirect():1339] redirect: SettingsConsole.REDIRECT
2022-05-11 04:37:45,452 INFO    MainThread:31249 [wandb_run.py:_redirect():1344] Redirecting console.
2022-05-11 04:37:45,452 INFO    MainThread:31249 [redirect.py:install():213] install start
2022-05-11 04:37:45,453 INFO    MainThread:31249 [redirect.py:install():228] install stop
2022-05-11 04:37:45,453 INFO    MainThread:31249 [redirect.py:install():213] install start
2022-05-11 04:37:45,454 INFO    MainThread:31249 [redirect.py:install():228] install stop
2022-05-11 04:37:45,454 INFO    MainThread:31249 [wandb_run.py:_redirect():1390] Redirects installed.
2022-05-11 04:37:45,454 INFO    MainThread:31249 [wandb_init.py:init():542] run started, returning control to user process
2022-05-11 04:37:50,510 INFO    MainThread:31249 [wandb_run.py:_atexit_cleanup():1446] got exitcode: 0
2022-05-11 04:37:50,512 INFO    MainThread:31249 [wandb_run.py:_restore():1418] restore
2022-05-11 04:37:50,512 INFO    MainThread:31249 [redirect.py:uninstall():232] uninstall start
2022-05-11 04:37:50,513 INFO    MainThread:31249 [redirect.py:_stop():287] _stop: stdout
2022-05-11 04:37:50,513 INFO    MainThread:31249 [redirect.py:_stop():293] _stop closed: stdout
2022-05-11 04:37:50,513 INFO    stdout    :31249 [redirect.py:_pipe_relay():129] relay done saw last write: stdout
2022-05-11 04:37:50,514 INFO    stdout    :31249 [redirect.py:_pipe_relay():145] relay done done: stdout
2022-05-11 04:37:50,515 INFO    MainThread:31249 [redirect.py:_stop():299] _stop joined: stdout
2022-05-11 04:37:50,515 INFO    MainThread:31249 [redirect.py:_stop():301] _stop rd closed: stdout
2022-05-11 04:37:50,515 INFO    MainThread:31249 [redirect.py:uninstall():236] uninstall done
2022-05-11 04:37:50,516 INFO    MainThread:31249 [redirect.py:uninstall():232] uninstall start
2022-05-11 04:37:50,516 INFO    MainThread:31249 [redirect.py:_stop():287] _stop: stderr
2022-05-11 04:37:50,516 INFO    MainThread:31249 [redirect.py:_stop():293] _stop closed: stderr
2022-05-11 04:37:50,516 INFO    stderr    :31249 [redirect.py:_pipe_relay():129] relay done saw last write: stderr
2022-05-11 04:37:50,517 INFO    stderr    :31249 [redirect.py:_pipe_relay():145] relay done done: stderr
2022-05-11 04:37:50,517 INFO    MainThread:31249 [redirect.py:_stop():299] _stop joined: stderr
2022-05-11 04:37:50,517 INFO    MainThread:31249 [redirect.py:_stop():301] _stop rd closed: stderr
2022-05-11 04:37:50,518 INFO    MainThread:31249 [redirect.py:uninstall():236] uninstall done
2022-05-11 04:37:51,897 INFO    MainThread:31249 [wandb_run.py:_wait_for_finish():1568] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 1207
  total_bytes: 1207
}

2022-05-11 04:37:53,904 INFO    MainThread:31249 [wandb_run.py:_wait_for_finish():1568] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 2343
  total_bytes: 2343
}

2022-05-11 04:37:54,977 INFO    MainThread:31249 [wandb_run.py:_show_files():1790] logging synced files
